### PR TITLE
TSE Review.

### PR DESCRIPTION
**Pull request by carlosveigadev**
Added all the elements and style for the Page applying all the theories learned about the position and floating elements. Including float, box model, flexbox, CSS reset. We also made the recommended validations with Linter and W3 Validator.

**Navigation Bar**

- Defined the fixed position
- Included Buttons, Logo, and Links

**First Article, Second Article and Third Article.**

- Included Media Icons
- Added the paragraphs
- Embedded images and video
- Position the section and its elements
- Style the text

**Explanation Grid**

- Added the images
- Added the paragraphs

**Asides**

- Position the aside using float
- Position of the Elements using flex
- Added the paragraphs

**Footer**

- Styled and positioned the footer list 
- Positioned all the list with its links
- Included the borders

### CODE REVIEW

File: begin.css [hunk 1]
```diff
@@ -0,0 +1,482 @@
+/* General */
```
Comment (lucasmazo32) [inline begin.css]: - [x] Elements should be aligned with flex, grid, and float. For this project you only use flex and float. Please align items with grid. 
> image

File: begin.html [hunk 2]
```diff
@@ -0,0 +1,792 @@
...
+          <li><a href="#">Your Ad Choices</a></li>
+          <li><a href="#">Privacy</a></li>
+          <li><a href="#">Terms of Service</a></li>
+          <li><a href="#">Terms of Sale</a></li>
+          <li><a href="#">Site Map</a></li>
+          <li><a href="#">Help</a></li>
+          <li><a href="#">Subscriptions</a></li>
+        </ul>
+      </div>
+    </footer>
```
Comment (lucasmazo32) [inline begin.html]: - [x] There is a part of the page that sticks to the bottom. Please add it.

> image

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

For the project to get approved, kindly implement the changes mentioned below.

### Happy coding! :computer:

## Lucas Mazo

File: begin.css (hunk 1 above)
Comment (carlosveigadev) [reply begin.css]: This requirement was done as asked.

File: begin.html (hunk 2 above)
Comment (carlosveigadev) [reply begin.html]: This requirement was done as asked.

**Comment by carlosveigadev** [comment]
@lucasmazo32 

- We added the grid element to the inflation explanation;
- We added the bottom fixed menu;
- We also added the suggestions that our team gave us to improve the project.

**Review by trekab — APPROVED** [review]
# Approved!
Hi @wrakc and @epinczinger, you have done a great job! :clap: You may merge to master, submit the project completion form, and continue to your next project.

_Happy coding!_ :wink:
__Treasure Kabareebe__

**Pull request merged**